ush/config_defaults.yaml

```diff
@@ -857,6 +857,7 @@ workflow:
   FCST_LEN_HRS: 24
   FCST_LEN_CYCL:
     - '{{ FCST_LEN_HRS }}'
+  NUM_FCST_LEN_CYCL: 1
```

Comment:
As suggested above, this may be able to be removed given that the information is already available by interrogating `FCST_LEN_CYCL`.

Reply:
Agreed. I'll remove it soon.